Zewa/Dependency.php

```diff
@@ -56,18 +58,27 @@ public function resolve($class, $persist = false)
             return $this->getDependency($class);
         }
 
-        $reflectionClass = new \ReflectionClass($class);
-        $constructor = $reflectionClass->getConstructor();
+        try {
+            $reflectionClass = new \ReflectionClass($class);
 
-        if (! $constructor || $constructor->getParameters() === 0) {
-            $dependency = new $class;
-            return $this->load($class, $dependency, $persist);
-        }
+            $constructor = $reflectionClass->getConstructor();
+
+            if (!$constructor || $constructor->getParameters() === 0) {
+                $dependency = new $class;
+                return $this->load($class, $dependency, $persist);
+            }
 
-        $params = $this->constructConstructorParameters($reflectionClass);
-        $dependency = $reflectionClass->newInstanceArgs($params);
+            $params = $this->constructConstructorParameters($reflectionClass);
+            $dependency = $reflectionClass->newInstanceArgs($params);
 
-        return $this->load($class, $dependency, $persist);
+            return $this->load($class, $dependency, $persist);
+        } catch (\Exception $e) {
+            echo "<PRE>";
```

Comment:
Why in the world would you want to catch and echo an exception when that is essentially what it does on it's own when it's not caught?   This doesn't seem like something that should be in a framework.   print_r and var_dump  ?